coral/docs/development-with-local-klaw.md

```diff
@@ -0,0 +1,78 @@
+# Development with local Klaw
+
+ℹ️ The proxy is a work in progress.
+
+Please check out the [proxy README](../proxy/README.md) for more detailed information on how to use the proxy in your development process. You'll also find explanation and workaround for the things that are currently not working.
+
+## Table of content
+
+* [Basic setup](#basic-setup)
+* [First setup](#first-setup)
+* [How to run it](#how-to-run-it)
+
+ 
+## Basic setup
+
+ℹ️ environment settings are located in the file [`.env.local-api`](../../coral/.env.local-api).
+
+** ℹRequirements**
+
+- [node](https://nodejs.org/en/) needs to be installed <br/>
+  -> see [nvmrc](../.nvmrc) or the `engines` definition in [package.json](../package.json) for version).
+- Coral uses [pnpm](https://pnpm.io/) (version 7) as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm.
+
+1. navigate to [`/coral`](../../coral)
+2. run `pnpm install`
+3. run `pnpm add-precommit` the first time you install the repository to set the custom directory for our pre commit hooks.
+- go to directory [`coral/proxy`](../../coral/proxy)
+- run `pnpm install` there, too
+4. Run development:
+   4.1. If you have not setup the docker environments, please follow [First setup]()
+   4.2. If you already have a setup, run `pnmv dev:[start|restart]`
```

Comment:
🤔  one builds and starts and the other only starts. I thought that's clear :D especially because it also links to the proxy readme. 

I'll be more specific!